scripts/importers/companyList.js

```diff
@@ -5,11 +5,13 @@ global.companyList = function(listData){
      */
     var companyListLoc = 'https://raw.githubusercontent.com/mozilla-services/shavar-prod-lists/master/disconnect-blacklist.json';
     var remapDataLoc = 'https://raw.githubusercontent.com/mozilla-services/shavar-prod-lists/master/google_mapping.json'
-    var trackerList = {};
+    var trackerList = { TopTrackerDomains:{}};
     var trackerTypes = ['Advertising', 'Analytics', 'Disconnect', 'Social'];
     var request = require('request');
     var remapData, companyList;
     
+    let majorNetworks = {"Google":true, "Facebook":true, "Twitter":true, "Amazon":true, "AppNexus":true, "Oracle":true};
```

Comment:
you can get this from defaultSettings. https://github.com/duckduckgo/chrome-zeroclickinfo/blob/beta/data/defaultSettings.js

Reply:
is settings really the right place for that?

Reply:
either way, @jdorweiler  and I agreed on a separate discussion to ignore this for now as the settings file is js and declares a const. Could be changed but needs to be fixed in another PR.

Reply:
added a note in the source

Reply:
👍